modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -2,6 +2,20 @@
 :description: Provisioning cluster resources is managed at the collection or bucket level, depending upon the service affected.
 :navtitle: Provisioning Cluster Resources
 :page-aliases: ROOT:managing-clusters
+:page-toclevels: 2
+
+// API refs
+:bucket-api-reference: pass:q[BucketManager -- https://pkg.go.dev/github.com/couchbase/gocb/v2?tab=doc#Cluster.Buckets[`Cluster.buckets()`]]
```

Comment:
just a thought (I realise this is already in -common, so not requesting a change now) but might it be more useful to store the URL / URL + description separately from the "BucketManager --" text? hmm... well, not important anyway.